softdevice_controller/include/sdc.h

```diff
@@ -391,6 +402,14 @@ typedef union
      *  Default: See @ref sdc_cfg_adv_buffer_cfg_t.
      */
     sdc_cfg_adv_buffer_cfg_t adv_buffer_cfg;
+    /** Configures the maximum size of the Filter Accept List.
+     *
+     * Setting this value to 0 will configure the controller to
+     * use hardware-based filtering.
+     *
+     * Default: @ref SDC_DEFAULT_FAL_SIZE.
+     */
+    uint8_t fal_size;
```

Comment:
Should we make this a uint16_t instead?

Reply:
Sure, why not. Then I guess we'll never have to update it

Reply:
Can be done in a later PR